test/shared-class.test.js

```diff
@@ -29,11 +29,6 @@ describe('SharedClass', function() {
       expect(sc.http.path).to.equal('/some');
     });
 
-    it('fills http.path using a normalized path', function() {
-      var sc = new SharedClass('SomeClass', SomeClass, {normalizeHttpPath: true});
```

Comment:
In that case, let's prioritize model-level setting over app-level setting. In other words, if application sets `normalizeHttpPath: false`, but model sets `normalizeHttpPath: true`, then the path should be normalized.